src/components/delete-confirmation-modal.tsx

```diff
@@ -0,0 +1,65 @@
+import '@isrd-isi-edu/chaise/src/assets/scss/_export.scss';
+
+import Button from 'react-bootstrap/Button';
+import Modal from 'react-bootstrap/Modal';
+
+import DisplayValue from '@isrd-isi-edu/chaise/src/components/display-value';
+
+type DeleteConfirmationModalProps = {
+  /**
+   * prop to show modal
+   */
+  show: boolean;
+  /**
+   * prop to trigger on delete confirmation
+   */
+  onConfirm: (tuple: any, isRelatable?: boolean, isUnlink?: boolean) => void;
+  /**
+   * prop to trigger on cancel
+   */
+  onCancel: () => void;
+  /**
+   * tuple reference
+   */
+  tuple: any;
+  /**
+   * button label prop
+   */
+  buttonLabel: string;
+};
+
+/**
+ * returns Modal Component - Component that renders delete comfirmation dialog
+ */
+const DeleteConfirmationModal = ({ show, onConfirm, onCancel, tuple, buttonLabel }: DeleteConfirmationModalProps) => {
+  return (
+    <Modal show={show} onHide={onCancel} keyboard={false}>
```

Comment:
What's the reason for passing the keyboard prop? We want users to be able to cancel the modal by just pressing escape (and it already is working this way too).

Also while it won't make any difference, for consistently let's specify `size='sm'` here.